www/src/pages/components/chip/ios/index.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: Chip
+description: Compact controls that allow for toggling and filtering
+---
+
+import { graphql } from 'gatsby';
+import { ComponentHeader } from 'components/thumbprint-components';
+
+<ComponentHeader data={props.data} />
+
+### Usage
+
+Chips allow users to make selections, filter content, or trigger small actions. Chips should appear as a group of multiple elements in most cases. Unlike buttons, chips shouldn’t navigate away from the current page. If the action requires a new page to load, please use a button instead of a chip.
+
+### Components
+
+There are two types of chip components: ToggleChip and FilterChip.
+
+A toggle chip behaves the same way as a checkbox and should be used for a set of options where more than one option can be selected.
+
+![Toggle chip](/img/ios-toggle-pill.png)
```

Comment:
The filenames for these images should be `...-chip.png` not `...-pill.png` which is why they aren't showing up in the preview deploy